README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
```

Comment:
I recommend updating the rails generated readme with info about your project (even a copy paste of the project requirements), it's helpful for if you look back over projects later on